sw/device/tests/crypto/cryptotest/firmware/trigger_sca.c

```diff
@@ -27,7 +27,7 @@ status_t handle_trigger_sca_select_source(ujson_t *uj) {
   cryptotest_trigger_sca_source_t uj_trigger;
   TRY(ujson_deserialize_cryptotest_trigger_sca_source_t(uj, &uj_trigger));
 
-  sca_select_trigger_type((sca_trigger_type_t)uj_trigger.source);
+  sca_select_trigger_type(read_32(uj_trigger.source));
```

Comment:
I am a bit confused by this change: the [definition](https://github.com/lowRISC/opentitan/blob/0a0d043d44fd64a7bf9979634d60f9e833247f30/sw/device/tests/crypto/cryptotest/json/trigger_sca_commands.h#L24) of `trigger.source` is a `uin8_t[TRIGGERSCA_CMD_MAX_SOURCE_BYTES]` with `TRIGGERSCA_CMD_MAX_SOURCE_BYTES = 1` so this is really one byte. If you call `read_32`, it seems that you are accessing past the end of the array?

Reply:
Good point, I've changed the code to just read the retrieved 8-bit value.